setup.py

```diff
@@ -9,15 +9,15 @@
 
 setuptools.setup(
     name="databricks-rocket",
-    version="2.1.0",
+    version="2.1.1",
```

Comment:
Since this is a new feature, I think `2.2.0` would technically be correct. Not so important, though.